features/step_definitions/user_steps.rb

```diff
@@ -230,8 +232,8 @@
 end
 
 Then /^a user account should exist for "(.*?)"$/ do |login|
-   user = User.find_by(login: login)
-   assert !user.blank?
+  user = User.find_by(login: login)
+  assert user.present?
```

Comment:
Can you switch this to an RSpec expect instead? (We should remove minitest assertions at some point.)